cleansweep/views/admin.py

```diff
@@ -1,6 +1,6 @@
 """Views of the admin panel.
 """
-
+from cleansweep.plugins.committees.models import CommitteeType
```

Comment:
Yeah I was thinking about that too. Can import it inside the function and also check if the plugin is enabled before doing anything.